### DISCUSSION TITLE
支持建立多个站点

### DISCUSSION THREAD

**Discussion by xiandafu** [vuepress-theme-hope/vuepress-theme-hope#3955] [Q&A] [answered]
### Checklist

- [X] I searched existing issues and no one else requests similar feature.
- [X] I think that 25%+ users are positive towards this feature.

### Describe the feature

希望 vuepress 能像apach那样建立多个站点。这是因为我有俩个开源对应了俩个站点（在线文档），我并不想放在一起作为一个站点。
我现在想到的是设置俩个doc目录，并分别启动

请问有什么办法能做到？

**Comment by Mister-Hope** [comment]
有什么阻止你这样做？

**Comment by Mister-Hope** [comment, marked as answer]
建立若干子文件夹，分别使用vuepress dev 文件夹名称 和 vuepress build 文件夹名称来操作不同文件夹。
